src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -75,4 +97,71 @@ class LocalBackend extends Backend {
       taskOutput.name -> rawValue.map { possiblyAutoConvertedValue(taskOutput, _) }
     }.toMap
   }
+
+  /**
+   * Creates host execution directory, inputs path, and outputs path.  Stages any input files into the workflow-inputs
+   * directory and localizes their paths relative to the container.
+   */
+  override def initializeWorkflow(workflow: WorkflowDescriptor, actualInputs: Map[FullyQualifiedName, WdlValue]): Map[FullyQualifiedName, WdlValue] = {
+    val hostExecutionDirectory = hostExecutionPath(workflow).toFile
+    hostExecutionDirectory.mkdirs()
+
+    val hostExecutionAbsolutePath = hostExecutionDirectory.getAbsolutePath
+    val hostInputsPath = Paths.get(hostExecutionAbsolutePath, "workflow-inputs")
+    hostInputsPath.toFile.mkdir()
+    val hostOutputsPath = Paths.get(hostExecutionAbsolutePath, "workflow-outputs")
+    hostOutputsPath.toFile.mkdir()
+    stageAndLocalizeWorkflowInputs(workflow, actualInputs)
+  }
+
+  /**
+   * {{{cromwell-executions + three-step + 0f00-ba4 = cromwell-executions/three-step/0f00-ba4}}}
+   */
+  private def hostExecutionPath(workflow: WorkflowDescriptor): Path = Paths.get("cromwell-executions", workflow.name, workflow.id.toString)
+
+  /**
+   * {{{/root + cromwell-executions/three-step/0f00-ba4 + call-ps + stdout.out = /root/cromwell-executions/three-step/0f00-ba4/call-ps/stdout.out}}}
```

Comment:
I'm pretty sure the right-hand-side of this equation can be shorter.  In fact why not just `/root/stdout.out`?  Unless I'm missing something, for 4 directory levels there will be only a directory that contains exactly 1 other directory.